### PR TITLE
fix moreUserTags

### PR DESCRIPTION
**Pull request by sunnniee**

### CODE REVIEW

File: src/plugins/moreUserTags/index.tsx [hunk 1]
```diff
@@ -230,10 +228,10 @@ export default definePlugin({
         },
         // in the member list
         {
-            find: ".renderBot=function(){",
+            find: ".Messages.GUILD_OWNER,",
             replacement: {
-                match: /\.BOT;return null!=(\i)&&.{0,10}\?(.{0,50})\.botTag,type:\i/,
-                replace: ".BOT;var type=$self.getTag({...this.props,origType:$1.bot?0:null,location:'not-chat'});return type!==null?$2.botTag,type"
+                match: /(.USE_EXTERNAL_EMOJIS,.{0,50}(?<channel>\i)\),\(0,.{0,350})(?<type>\i)=\(null==(?<user>\i).{0,50}\.BOT,(.{0,10})&&\i.bot/,
```
Comment (Vendicated) [inline src/plugins/moreUserTags/index.tsx:233]: this has way too many wildcards. why do you match this much code? that's most likely not needed

Comment (sunnniee) [reply src/plugins/moreUserTags/index.tsx]: it was indeed not needed 👍

**Comment by sunnniee** [comment]
so is this getting merged or

**Comment by Vendicated** [comment]
thank you

**Pull request merged**